src/views/MainPage/mainPage.js

```diff
@@ -0,0 +1,74 @@
+import './mainPage.css';
+
+export function renderMainPage() {
+  renderMainView();
+  renderAnimalFact();
+}
+
+function renderMainView() {
```

Comment:
This function is quite long but we will leave it for now.  It will be reworked in the future